src/components/Jobs/Card/card.css

```diff
@@ -0,0 +1,57 @@
+.grid-container {
```

Comment:
Was there a specific reason to create a custom implementation of a grid vs using the grid that is already defined? If not please remove this in favor of gird use throughout the app.

Reply:
I changed the css and the jsx for the cards. We are using panel classes from bootstrap 3 now